src/Sarif.FunctionalTests/ConverterTestData/PREfast/AA_IndexAlias.cpp.xml.sarif

```diff
@@ -9,22 +9,7 @@
       },
       "files": {
         "file:///d:/src/sarif-sdk/src/sarif.functionaltests/convertertestdata/prefast/src/advancedaliasing/aa_indexalias.cpp": {
-          "mimeType": "text/x-cpp",
-          "contents": "Ly8gVHJhY2sgImludGVyZXN0aW5nIiB2YXJpYWJsZXMgYW5kIGRldGVjdCB3aGVuIHRoZXkgYXJlIGFsaWFzZWQuDQoNCnZvaWQgQzYzODVfSW5kZXhBbGlhcyhpbnQgaW5wdXRJbmRleCkNCnsNCglpbnQgKiBjb250ZW50U3RvcmVzID0gbmV3IGludCBbMl07DQoNCgljb250ZW50U3RvcmVzWzBdID0gMDsNCg0KCWNvbnRlbnRTdG9yZXNbMV0gPSAxOw0KDQoJd2hpbGUgKGlucHV0SW5kZXggPiAxKQ0KCXsNCgkJaW50IGluZGV4ID0gaW5wdXRJbmRleDsNCg0KCQljb250ZW50U3RvcmVzWzBdID0gY29udGVudFN0b3Jlc1tpbmRleF07DQoJfQ0KDQoJZGVsZXRlIFtdIGNvbnRlbnRTdG9yZXM7DQp9DQo=",
```

Comment:
A bunch of these probably need to be fixed.  What is the significance of this exactly?  All the tests pass without this being present.